### ISSUE TITLE
Azure Relay Hybrid Connection 

### ISSUE THREAD

**Issue by bradwilliamson** [terraform-providers/terraform-provider-azurerm#1320]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Add the ability to create & delete Hybrid Connection child resource for the new Azure Relay namespace released in 1.6.0.  

What they are: 
https://docs.microsoft.com/en-us/azure/app-service/app-service-hybrid-connections

### New or Affected Resource(s)

Azure Relay namespace 

* azurerm_XXXXX

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

* #0000

**Comment by thomassampson** [comment]
👍

**Comment by mwalton-esendex** [comment]
This appears to have been released in 1.38.0 (according to the changelog), but is not documented on the website, although I note the Markdown file has been written and added to the repository.
